app/models/setting.rb

```diff
@@ -13,4 +13,14 @@ def self.get(key)
     value_object = Setting.where(key: key, status: "Active").first
     value_object.value if value_object
   end
+
+  def active?
+    self.status == 'Active'
+  end
+
+  def self.get_value(key, fallback_value = nil)    
+    value = get(key)
+    (Integer value rescue value) || fallback_value
```

Comment:
I'm not familiar with that form. Is that the same as `value.to_i` ?

Reply:
Yes but `to_i` converts nil to 0, so I used this. If the value cannot be converted to the integer the raised exception will rescued and will return the actual value, if the value is nil, fallback value will be returned. 
Is this ok ?

Reply:
And now as we are sure that this method will only return integer, I think this can be changed to 
`(Integer value rescue nil) || fallback_value`. 
And if possible we also need to add validation to prevent non-integer value for this setting in the first place.

Reply:
I would appreciate a brief comment here, for others following along. Basically close to your comment above. I like to `resque nil` version
For validation, have a `set_integer` method ?

Thanks for your help !

Reply:
Sorry @kgf, what do you want me to comment about ?